src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx

```diff
@@ -1,20 +1,59 @@
 import { Box, BoxProps, Input } from "@artsy/palette"
 import { useFormikContext } from "formik"
+import { useEffect } from "react"
+import { useRouter } from "v2/System/Router/useRouter"
+import { ContactInformation_me } from "v2/__generated__/ContactInformation_me.graphql"
+import { useSubmission } from "../../Utils/useSubmission"
 
+export const getContactInformationFormInitialValues = (
+  me: ContactInformation_me
+) => ({
+  name: me?.name || "",
+  email: me?.email || "",
+  phone: me?.phone || "",
+})
 export interface ContactInformationFormModel {
   name: string
   email: string
   phone: string
 }
 
-export interface ContactInformationFormProps extends BoxProps {}
+export interface ContactInformationFormProps extends BoxProps {
```

Comment:
is there any reason why form props extend BoxProps?

Reply:
To be able manage `ContactInformationForm` like this (by any box props): https://github.com/artsy/force/blob/8259853334cff10c71753923d20333f8e9cea084/src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/ContactInformation.tsx#L107

Reply:
oh I see! 

**non-blocking**: in this case, my suggestion would be to control the spacing in the parent, something like the below. it'll increase the readability, and we can get rid of passing/spreading the props while we can. 

```
<Box my={6}>
     <ContactInformationForm me={me} />
</Box>
```

Reply:
This is an existing pattern in Force to avoid unnecessary wrappers.